source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,320 @@ Sorts Builders
+class is the sort criteria builder provided by the MongoDB Java Driver.
+``Sorts`` provides static factory methods for all
+sort criteria operators supported by MongoDB. Each method returns
+an instance of the
+:java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html?>`
+type. 
+
+The following examples show you how to use the static factory
+methods provided by the ``Sorts`` class to sort your queries.
+The examples use a sample collection ``sort_example`` that contains
+the following documents:
+
+.. code-block:: none
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Direction
+-----------------
+
+The ``Sorts`` class provides two methods for specifying the sort order of data
+that does not contain any duplicate values. Data can be sorted from smallest to largest 
+with ``Sorts.ascending()``, and data can be sorted from largest to smallest with
+``Sorts.descending()``. The following two subsections provide examples
+on how to use these two methods. 
+
+Ascending
+~~~~~~~~~
+
+To specify an ascending sort, use the ``Sorts.ascending()``
+static factory method. An ascending sort returns 
+your documents from smallest to largest on the field name specified.
+
+The ``ascending`` method can be used as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts
+
+   // <MongoCollection setup code here>
+
+   collection.find().sort(Sorts.ascending("<field name>"))
+
+This will return a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`
+object containing the documents in your collection sorted 
+from smallest to largest on the specified field name. 
+
+.. example::
+
+   In this code example, we use ``ascending`` to
+   sort the ``sort_example`` collection by ``"_id"``:
+
+   .. code-block:: java
+   
+      import static com.mongodb.client.model.Sorts.ascending;
+   
+      // <MongoCollection setup code here>
+      
+      Bson id_sort = ascending("_id");
+      List<Document> results = new ArrayList<>();
+      collection.find().sort(id_sort).into(results);
+      for (Document result : results) {
+          System.out.println(result.toJson());
+      }
+
+   The output of the above code snippet should look something like this: 
+   
+   .. code-block:: none
+   
+      {"_id": 1, "letter": "c", "food": "coffee with milk"}
+      {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+      {"_id": 3, "letter": "a", "food": "maple syrup"}
+      ...
+   
+An important thing for you to keep in mind is that MongoDB does
+not guarantee sort order for documents that have fields with identical values.
+This means that ``ascending("letter")`` could have any of these
```

Comment:
I would avoid this type of notation in which you're showing what you're passing to a method outside of a code snippet. Instead you can convey the same meaning with something like:
"This means that if you pass "letter" to the ascending() method, the first document returned could be any of the following documents:"

Reply:
Thanks for the note. Fix for occurrences of this pattern on my page: https://github.com/mongodb/docs-java/pull/59/commits/bd8312cb77e520c0f2a987dfdc4a7e06e4592c0c

Going forward, I will avoid ``"<some string>"`` outside of code.